# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../softdog/softdog.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


define _GNU_SOURCE (for extended glibc features).

 - Clifford

--- ./Makefile~	Mon May 11 22:14:00 1998
+++ ./Makefile	Wed Sep 23 00:47:44 1998
@@ -10,7 +10,7 @@
 #DEBUG	= -g -DDEBUG
 CFLAGS	= -Wall $(DEBUG) $(INCLUDE) $(OPTIMIZE)\
           -DDEVICE=\"$(DEVICE)\" -DMY_MAJOR=$(MAJOR) -DMY_MINOR=$(MINOR)\
-          -DBINNAME=\"$(BINNAME)\"
+          -DBINNAME=\"$(BINNAME)\" -D_GNU_SOURCE
 
 # Installation paths
 BINDIR	=/sbin
